oaknational/Oak-Web-Application
# feat: search filter tracking [LESQ-541]

## Description

Music year: {owa_music_year}

- use all active filters in the events with the 'search filter option selected' field, not just keystages

![Screenshot 2024-01-09 at 11 52 03](https://github.com/oaknational/Oak-Web-Application/assets/45038878/89a3f407-26e4-4aac-99f0-224390f964aa)

## How to test

1. Go to https://deploy-preview-2166--oak-web-application.netlify.thenational.academy
2. Search something and apply filters
3. You should see the event come though with all selected filters

## Checklist

- [ ] Any event that has the “Search filter option selected” reports all the filters that are selected at the time of the search event being sent
